gcc/config/riscv/riscv.md

```diff
@@ -1064,7 +1064,7 @@
   [(set (match_operand:DI     0 "register_operand"     "=r,r")
 	(zero_extend:DI
 	    (match_operand:SI 1 "nonimmediate_operand" " r,m")))]
-  "TARGET_64BIT && !TARGET_BITMANIP"
+  "TARGET_64BIT && !(TARGET_ZBB || TARGET_ZBP)"
```

Comment:
This is actually addu.w now which is ZBB and ZBA.  Not ZBP.  But ZBP could be supported with an optional assembler change.

Reply:
Thanks for catching this! fixed!